angular.json

```diff
@@ -37,7 +37,11 @@
             ],
             "styles": [
               "src/theme.less",
-              "src/styles.scss"
+              "src/styles.scss",
+              "./node_modules/primeicons/primeicons.css",
+              "./node_modules/primeflex/primeflex.css",
```

Comment:
I think we need to add these files above our custom styles file.

Reply:
ok.